src/functions-and-arrays.js

```diff
@@ -1,19 +1,61 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  
+  if(num1 < num2){
+    return num2;
+  }
+  else if (num1 > num2){
+    return num1;
+  }
+  else {
+    return num1 ;
+  }
 
+}
 
 
-// Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+// Iteration #2: Find longest word
+const arr = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+
+function findLongestWord(arr) {
```

Comment:
This got a bit confusing. A better way to write this function would be something like this:

```
function findLongestWord(arr) {
  let longWord = "";
  if (!arr.length) {
    return null;
  } else if (arr.length === 1) {
    return arr[0];
  } else {
    for (let i = 0; i < arr.length; i++) {
    if (arr[i].length > longWord.length) {
      longWord = arr[i]
    } 
  } return longWord; 
}
}
```